### ISSUE TITLE
Automatic PyPI releases

### ISSUE THREAD

**Issue by SushiWaUmai** [darealvenum/venumLang2#21]
It'd be nice if PyPI releases would happen more frequently, therefore I suggest that we make a simple Github Actions workflow that handles the publishing.

**Comment by farkon00** [comment]
> It'd be nice if PyPI releases would happen more frequently, therefore I suggest that we make a simple Github Actions workflow that handles the publishing.

When do you want to publish it. On which event?
Most common is on release, but this repo has no releases. On commit is too much, on merge isntt a good idea, venum does work in main branch, so his changes will not affect that and or he waits for merge or he publishes release himself.

**Comment by SushiWaUmai** [comment]
I think we should use the Github Flow development workflow, just make a dev branch and several feature branches and publish on push onto main. This forces us to use the correct conventions.

**Comment by farkon00** [comment]
Yes, this sounds like good approach for developemnet. I can make this action in next pr. Probably will add tests too.
